imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml

```diff
@@ -1575,901 +1575,38 @@ lo-ialirt-fe_hiq:
-    CATDESC: Priority 6 PHA Type
-    DEPEND_1: event_num
-    FIELDNAM: Priority 6 PHA Type
-    LABL_PTR_1: event_num_label
-    VALIDMAX: 4
-
-p6_position:
-    <<: *direct_events
-    CATDESC: Priority 6 Position
-    DEPEND_1: event_num
-    FIELDNAM: Priority 6 Position
-    LABL_PTR_1: event_num_label
-    VALIDMAX: 32
-
-p6_spin_sector:
-    <<: *direct_events
-    CATDESC: Priority 6 Spin Angle
-    DEPEND_1: event_num
-    FIELDNAM: Priority 6 Spin Angle
-    LABL_PTR_1: event_num_label
-    VALIDMAX: 128
-
-p6_tof:
-    <<: *direct_events
-    CATDESC: Priority 6 Time of Flight
-    DEPEND_1: event_num
-    FIELDNAM: Priority 6 Time of Flight
-    FILLVAL: 65535
-    LABL_PTR_1: event_num_label
-    VALIDMAX: 1024
-
-p7_apd_id:
-    <<: *direct_events
-    CATDESC: Priority 7 APD ID
-    DEPEND_1: event_num
-    FIELDNAM: Priority 7 APD ID
-    LABL_PTR_1: event_num_label
-    VALIDMAX: 32
-
-p7_apd_energy:
-    <<: *direct_events
-    CATDESC: Priority 7 APD Energy
-    DEPEND_1: event_num
-    FIELDNAM: Priority 7 APD Energy
-    FILLVAL: 65535
-    LABL_PTR_1: event_num_label
-    VALIDMAX: 512
-
-p7_gain:
-    <<: *direct_events
-    CATDESC: Priority 7 APD Gain
-    DEPEND_1: event_num
-    FIELDNAM: Priority 7 APD Gain
-    LABL_PTR_1: event_num_label
-    VALIDMAX: 1
-
-p7_data_quality:
-    <<: *direct_events
-    CATDESC: Priority 7 Data Quality
-    FIELDNAM: Priority 7 Data Quality
-    LABLAXIS: " "
-    VALIDMAX: 1
-
-p7_energy_step:
-    <<: *direct_events
-    CATDESC: Priority 7 Energy Step
-    DEPEND_1: event_num
-    FIELDNAM: Priority 7 Energy Step
-    LABL_PTR_1: event_num_label
-    VALIDMAX: 128
-
-p7_multi_flag:
-    <<: *direct_events
-    CATDESC: Priority 7 Multi Flag
-    DEPEND_1: event_num
-    FIELDNAM: Priority 7 Multi Flag
-    LABL_PTR_1: event_num_label
-    VALIDMAX: 1
-
-p7_num_events:
-    <<: *direct_events
-    CATDESC: Priority 7 Number of Events
-    FIELDNAM: Priority 7 Number of Events
-    FILLVAL: 65535
-    LABLAXIS: " "
-
-p7_type:
-    <<: *direct_events
-    CATDESC: Priority 7 PHA Type
-    DEPEND_1: event_num
-    FIELDNAM: Priority 7 PHA Type
-    LABL_PTR_1: event_num_label
-    VALIDMAX: 4
-
-p7_position:
-    <<: *direct_events
-    CATDESC: Priority 7 Position
-    DEPEND_1: event_num
-    FIELDNAM: Priority 7 Position
-    LABL_PTR_1: event_num_label
-    VALIDMAX: 32
-
-p7_spin_sector:
-    <<: *direct_events
-    CATDESC: Priority 7 Spin Angle
-    DEPEND_1: event_num
-    FIELDNAM: Priority 7 Spin Angle
-    LABL_PTR_1: event_num_label
-    VALIDMAX: 128
-
-p7_tof:
-    <<: *direct_events
-    CATDESC: Priority 7 Time of Flight
-    DEPEND_1: event_num
-    FIELDNAM: Priority 7 Time of Flight
-    FILLVAL: 65535
-    LABL_PTR_1: event_num_label
-    VALIDMAX: 1024
+# Minimum attrs setting for DE data
+de_2d_attrs:
+  CATDESC: Direct event data
+  FIELDNAM: Direct Event Data
+  LABLAXIS: Values
+  DEPEND_0: epoch
+  DEPEND_1: priority
+  FILLVAL: -9223372036854775808
```

Comment:
If I recall correctly, some of the fields had differing FILLVALS from others according to Joey's validation data. I am not sure how important that is to distinguish.